src/firebolt/async_db/cursor.py

```diff
@@ -584,6 +612,49 @@ def setinputsizes(self, sizes: List[int]) -> None:
     def setoutputsize(self, size: int, column: Optional[int] = None) -> None:
         """Set a column buffer size for fetches of large columns (does nothing)."""
 
+    @check_not_closed
+    async def cancel(self, query_id: str) -> None:
+        """Cancel a server-side async query."""
+        try:
+            resp = await self._api_request(
+                parameters={"query_id": query_id}, url_suffix="cancel"
+            )
+            if resp.status_code == codes.BAD_REQUEST:
+                raise OperationalError(
+                    f"Asynchronous query {query_id} was not cancelled."
+                )
+        except Exception:
+            self._state = CursorState.ERROR
+            raise
+
+    @check_not_closed
+    async def get_status(self, query_id: str) -> QueryStatus:
+        """Get status of a server-side async query. Return the state of the query."""
+        try:
+            resp = await self._api_request(
+                # output_format must be empty for status to work correctly.
+                parameters={"query_id": query_id, "output_format": ""},
```

Comment:
Since we're deleting the key "output_format" in `_api_request`, wouldn't it make sense to not provide it in the first place?

Reply:
The thing is that it has a default value, so if it's missing the default needs to be subbed in. I agree that this isn't clean though. I'll give this some thought.

Reply:
Ah I get it now. Since `_api_request` is an internal function, can we add a check for no output format from user input on outside of it, in whichever function calls it? Then when we need to call `_api_request` with no output format we can call it as is.

Reply:
There actually never was a test before. It was always `JSONCompact`, which is what caused the problem. I added this functionality only for `get_status()`, which is why this solution isn't great. 

I also, just now, went ahead and changed `cancel()` so it removes it, and also when validating a `SET` parameter.

Reply:
This has been resolved.

Reply:
@ericf-firebolt This is what I meant by moving it outside, https://github.com/firebolt-db/firebolt-python-sdk/pull/196

Reply:
Ah, got it. Yeah, that's much cleaner. I feel like there's a reason I didn't to it, but it was a mistake not to. Anyway, that's merged.